src/map/packets/macroequipset.cpp

```diff
@@ -34,7 +34,7 @@ CAddtoEquipSet::CAddtoEquipSet(uint8* orig)
     // Should Push 0x116 (size 68) in responce
     // 0x04 is start, contains 16 4 byte parts repersently each slot in order
     this->setType(0x16); // TODO
```

Comment:
Is this one of my TODOs to change this to 116?

Reply:
Yeah, the ones I left are prefixed with  `// TODO: Verify`